src/core/aztec/decoder/Decoder.ts

```diff
@@ -232,7 +117,7 @@ export default class Decoder {
                 }
                 for (let charCount = 0; charCount < length; charCount++) {
                     if (endIndex - index < 8) {
-                        index = endIndex; // Force outer loop to exit
+                        index = endIndex;  // Force outer loop to exit
```

Comment:
If there's some setting for double spaces for inserting comments at the EOL, we should change it to single space to match Java.